core/py/src/braintrust_core/score.py

```diff
@@ -20,7 +20,7 @@ def as_dict(self):
         }
 
     def __post_init__(self):
-        if self.score < 0 or self.score > 1:
+        if self.score is not None and (self.score < 0 or self.score > 1):
```

Comment:
Can we change the type signature of score to `Optional[float]`?

Reply:
yep